test/services/asset_import/base.py

```diff
@@ -0,0 +1,90 @@
+import shutil
+import tempfile
+from test.services.asset_import.mocks import mocked_auth
+from test.utils import LocalDownloader
+from unittest import TestCase
+from unittest.mock import call
+
+from kili.graphql.operations.asset.mutations import (
+    GQL_APPEND_MANY_FRAMES_TO_DATASET,
+    GQL_APPEND_MANY_TO_DATASET,
+)
+from kili.services.asset_import import import_assets
+
+
+class ImportTestCase(TestCase):
+    def setUp(self):
+        self.project_id = "project_id"
+        self.test_dir = tempfile.mkdtemp()
```

Comment:
you can use a `with TemporaryDirectory() as tmp_dir` context, so you can get rid of the setup and teardown steps, and have a separated environment per test case